LICENSE

```diff
@@ -0,0 +1,233 @@
+                                 Apache License
```

Comment:
license file,  please keep the consistent with apache carbondata(https://github.com/apache/carbondata/blob/master/LICENSE)

Reply:
ok, i will fix it